INN/umbrella-sfpublicpress
# Update colors of global navbar

## Changes

This pull request makes the following changes:

- Updates the colors in the navbar according to the screenshot in #75 
- Updates the social icon colors in the footer to match the new colors in the header

![Screen Shot 2020-05-28 at 2 36 40 PM](https://user-images.githubusercontent.com/18353636/83179935-afc8f780-a0f0-11ea-8d09-25fdd0adbd4e.png)

Hover states:
![ezgif-5-46d69c6f6ce9](https://user-images.githubusercontent.com/18353636/83179941-b0fa2480-a0f0-11ea-8d9d-f73400b958ea.gif)

## Why



For #75

## Testing/Questions

Features that this PR affects:

- Global navbar styling
- Footer social icon styling


Questions that need to be answered before merging:

- [ ] Do all of the new colors look ok?

Steps to test this PR:

1. View the navbar and make sure all of the colors match the mockup screenshot in #75
